dir.props

```diff
@@ -502,32 +502,6 @@
     <SkipTests Condition="'$(SkipTests)'=='' and ('$(OsEnvironment)'=='Windows_NT' and '$(TargetsWindows)'!='true' and '$(OSGroup)'!='AnyOS')">true</SkipTests>
   </PropertyGroup>
 
-  <!-- *** start WCF Content *** -->
```

Comment:
Question only: I presume this became unnecessary now and was deliberately removed (not accidentally removed by copying the file from CoreFx directly).  Is that correct?